docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -127,23 +126,23 @@ Once the NuGet packages are updated to recent versions, the `<PackageReference>`
 
 Once you understand the state of your project's NuGet dependencies, the next thing to consider is .NET Framework API dependencies. The [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md) tool is useful for understanding which of the .NET APIs your project uses are available on other .NET platforms.
 
-The tool comes as a [Visual Studio plugin](https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer), a [command-line tool](https://github.com/Microsoft/dotnet-apiport/releases), or wrapped in a [simple GUI](https://github.com/Microsoft/dotnet-apiport-ui), which simplifies its options. You can read more about using the .NET Portability Analyzer (API Port) via the GUI in [this blog post](https://devblogs.microsoft.com/dotnet/porting-desktop-apps-to-net-core/). If you prefer to use the command line, the necessary steps are:
+The tool comes as a [Visual Studio plugin](https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer), a [command-line tool](https://github.com/Microsoft/dotnet-apiport/releases), or wrapped in a [simple GUI](https://github.com/Microsoft/dotnet-apiport-ui), which simplifies its options. You can read more about using the .NET Portability Analyzer (API Port) using the GUI in the [Porting desktop apps to .NET Core](https://devblogs.microsoft.com/dotnet/porting-desktop-apps-to-net-core/) blog post. If you prefer to use the command line, the necessary steps are:
 
 1. Download the [.NET Portability Analyzer](https://github.com/Microsoft/dotnet-apiport/releases) if you don't already have it.
-1. Make sure the .NET Framework app to be ported builds successfully (this is a good idea prior to migration anyhow!).
-1. Run API Port with a command line like this:
+1. Make sure the .NET Framework app to be ported builds successfully (this is a good idea prior to migration regardless).
+1. Run API Port with a command line like this.
```

Comment:
I believe this should be colon as it was.

```suggestion
1. Run API Port with a command line like this:
```

Reply:
According to styling the code should be treated like a figure/table, not needing a `:` and really we should reserve it for other lists and bullet points. (I'm learning guidelines as it's being edited)

Reply:
I think most of the current articles uses the colon in such cases.